src/Core/Routing/Router.php

```diff
@@ -95,7 +95,6 @@ private function add(string $pattern, $callable, ?string $name, string $methode)
         if ($name) {
             $this->_nameRoute[$name] = $route;
         }
-        $this->baseMiddleware = null;
```

Comment:
You have to assume there could not be another problem while creating multiple groupe